openlibrary/templates/site/body.html

```diff
@@ -24,7 +24,7 @@
       </div>
       $# Paste next line into announcement variable to show the blue banner
       $# <strong>Title:</strong> Description &nbsp; <a href="#" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>
-      $ announcement = _("<strong>Together</strong>, let's build an Open Library for the World.") + ' <a class="cta-btn cta-btn--available" style="display: inline;" href="/sponsorship">Sponsor a Book</a>'
+      $ announcement = _("Parents, teachers, and students: Visit our new") + ' <a class="cta-btn cta-btn--available" style="display: inline;" href="/k-12">K-12 Student Library</a>'
```

Comment:
Let's i18n the whole sentence for better flow. @cclauss  I agree K-12 is a little US-centric, but we can make new collections for other nation's standards.

```suggestion
      $ announcement = _("Parents, teachers, and students: Visit our new <a %s>K-12 Student Library</a>", 'class="cta-btn cta-btn--available" style="display: inline;" href="/k-12"')
```